tests/test_target.py

```diff
@@ -14,17 +14,30 @@
 # limitations under the License.
 #
 
+''' Module for testing targets. '''
```

Comment:
Docstrings should generally be trimmed, no space at the beginning or the end (unless it's needed for indentation after a newline)